### ISSUE TITLE
Bibeltv.de video not downloading

### ISSUE THREAD

**Issue by das-auge** [yt-dlp/yt-dlp#8985]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Germany

### Provide a description that is worded well enough to be understood

I tried to download a video $ yt-dlp https://www.bibeltv.de/mediathek/videos/340415-gott-ist-nicht-tot-4-wir-sind-das-volk -vU and it did not work. I tried some other videos from the same site. They didn't download either. 

Here is the Dump file: https://gist.github.com/das-auge/3f2d672dd70a11b40eb8f784fefb1c33

The Mediathek holds many videos for only a short time and sometimes I do not manage to watch them, so it would be wonderful to save them for watching later. 

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['https://www.bibeltv.de/mediathek/videos/340415-gott-ist-nicht-tot-4-wir-sind-das-volk', '-vU']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version master@2024.01.09.031215 from yt-dlp/yt-dlp-master-builds [95e82347b] (zip)
[debug] Python 3.11.6 (CPython x86_64 64bit) - Linux-6.5.0-14-generic-x86_64-with-glibc2.38 (OpenSSL 3.0.10 1 Aug 2023, glibc 2.38)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.11.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.46.0, pyxattr-0.8.1, requests-2.31.0, secretstorage-3.3.3, sqlite3-3.42.0, urllib3-1.26.16, websockets-10.4
[debug] Proxy map: {}
[debug] Request Handlers: urllib
[debug] Loaded 1798 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp-master-builds/releases/latest
Latest version: master@2024.01.09.031215 from yt-dlp/yt-dlp-master-builds
yt-dlp is up to date (master@2024.01.09.031215 from yt-dlp/yt-dlp-master-builds)
[bibeltv:video] Extracting URL: https://www.bibeltv.de/mediathek/videos/340415-gott-ist-nicht-tot-4-wir-sind-das-volk
[bibeltv:video] 340415: Downloading webpage
ERROR: [bibeltv:video] 340415: Unable to extract next.js data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "/home/user/.local/bin/yt-dlp/yt_dlp/extractor/common.py", line 718, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.local/bin/yt-dlp/yt_dlp/extractor/bibeltv.py", line 124, in _real_extract
    self._search_nextjs_data(self._download_webpage(url, crn_id), crn_id),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.local/bin/yt-dlp/yt_dlp/extractor/common.py", line 1685, in _search_nextjs_data
    self._search_regex(
  File "/home/user/.local/bin/yt-dlp/yt_dlp/extractor/common.py", line 1266, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
```

**Comment by geosone** [comment]
yes and it would eb great to also have the download playlist feature also so a cron can be created.
